packages/core/admin/admin/src/content-manager/pages/EditView/components/DocumentActions.tsx

```diff
@@ -573,16 +618,30 @@ const UpdateAction: DocumentActionComponent = ({ activeTab, id, model, collectio
             navigate({
               pathname: `../${collectionType}/${model}/${res.data.id}`,
             });
+          } else if (
+            'error' in res &&
+            isBaseQueryError(res.error) &&
+            res.error.name === 'ValidationError'
+          ) {
+            setErrors(formatValidationErrors(res.error));
           }
         } else if (id || collectionType === SINGLE_TYPES) {
-          await update(
+          const res = await update(
```

Comment:
minor thing, just something on top of my head, I saw some references to collection types, single types, or logic that we do have in strapi/utils or strapi/types. As this is is something that might be extended in the future (which kinds of data we do have), it might be a good idea to unify how we distinguish between each type.

Backend side we would normally pass down the `uid` of the content-type, and infer if it's a single type or a collection type one.

Reply:
the `collectionType` comes from the URL and the FE at the moment will redirect if it's not `single-types` or `collection-types` we could share the values, but the core idea is it is part of the URL 😄